rca/editorial/serializers.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework.fields import Field
+
+
+class EditorialTypeTaxonomySerializer(Field):
+    def to_representation(self, value):
+        return [{"title": v.type.title, "id": v.type.id} for v in value.all()]
```

Comment:
Could `v.type` ever be `None`, thinking of a potential `AttributeError`?

If so, you may be able to mitigate it with something like `value.filter(type__isnull=False)`